src/ansys/mechanical/core/embedding/app.py

```diff
@@ -297,8 +297,8 @@ def _share(self, other) -> None:
         they all point to the same underlying application
         object. Because of that, special care needs to be
         taken to properly share the state. Other will be
```

Comment:
they all point to the same underlying application object. As a result, you need to be careful to properly share the state.